.github/action_scripts/delegated_staking/delegated-staking.js

```diff
@@ -688,7 +694,7 @@ const checkUpdateDelegatedStake = async (urls, account, stakeHash, nodeId) => {
             nodeId,
             amount: originalStake.amount,
             tokenLockRef: originalStake.tokenLockRef,
-            // rewardAmount: originalStake.rewardAmount, // balance is transferred
+            rewardAmount: originalStake.rewardAmount, // balance is transferred
```

Comment:
More of a question / comment here, are we asserting that the rewardAmount remains EXACTLY the same during the update? So if another reward round runs before this assertion, the test will fail? 

It seems like a greater than equals is required? Or to verify the exact state at every ordinal to know that we've actually transferred correct amount. I could see this becoming flaky.

Reply:
It's possible this could become flaky but I think it's unlikely. We hit this endpoint as soon as we get a successful response from the gL0 that the stake was created which would be before the snapshot that will contain the new stake position has been created, then we recheck every second if not initially found. Somehow a timed snapshot with additional rewards would need to undergo consensus and update the state in less than a second to have this be greater than the transferred reward amount the first time we check.